kfserving/kfserving-install/base/statefulset.yaml

```diff
@@ -40,14 +40,12 @@ spec:
             fieldRef:
               fieldPath: metadata.namespace
         - name: SECRET_NAME
-          value: kfserving-webhook-server-secret
-        - name: ENABLE_WEBHOOK_NAMESPACE_SELECTOR
```

Comment:
let's keep this env, it is specifically turned on for kubeflow setup.

Reply:
Done

Reply:
Done